scripts/pre_commit_linter.py

```diff
@@ -881,8 +883,84 @@ def _check_html_directive_name(all_files):
     return summary_messages
 
 
+def _check_directive_scope(all_files):
+    """This function checks that all directives have an explicit
+    scope key and it should not be scope: true.
```

Comment:
I think we need directives to have explicit isolate scope i.e, it must have `scope: {}`. isn't it?

Reply:
Yep, that should be the aim.

Reply:
Done.